src/graph/dot.ts

```diff
@@ -1,11 +1,11 @@
 import { createSelector } from 'reselect';
-import { getTypeGraphSelector } from './type-graph';
+import { getTypeGraphSelector, getDisplayOptions } from './type-graph';
 import { stringifyWrappers } from '../introspection/';
 import * as _ from 'lodash';
 
-export const getDotSelector = createSelector(getTypeGraphSelector, getDot);
+export const getDotSelector = createSelector(getTypeGraphSelector, getDisplayOptions, getDot);
 
-function getDot(typeGraph): string {
+function getDot(typeGraph, displayOptions): string {
```

Comment:
@soneymathew IMO Passing `displayOptions` between function adds a lot of boilerplate. Can you move all functions(except `objectValues`) inside `getDot`:
```js
function getDot() {

  return (/* ... */);

  function nodeField() {
    // can access options
  }
}

Reply:
Thanks for the feedback, will update

Reply:
Thanks for the feedback, will update

Reply:
done